src/main/java/svc/models/Users.java

```diff
@@ -0,0 +1,48 @@
+package svc.models;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Entity
+public class Users {
```

Comment:
Class name of `Users` seems odd. What is this used for?

Reply:
Deleted this file.  It was accidentally left from working an example piece of code.